localstack/utils/aws/templating.py

```diff
@@ -53,7 +53,7 @@ def urlDecode(self, s):
         return unquote_plus(s)
 
     def escapeJavaScript(self, s):
-        return str(s).replace("'", r"\'")
+        return str(s).replace('"', r'\"')
```

Comment:
Not sure about this change - this could have some side effects, and potentially break existing client code. Should we escape both - single and double quotes here? Would be great if we can also add a small unit test to cover this functionality.

Reply:
Added both. There was an issue when the payload is JSON. It was not escaping the / for the double quote and `json.loads()` only accepts values in double-quotes only.